adaptive_breakpoints/lib/src/adaptive_breakpoints.dart

```diff
@@ -90,6 +91,98 @@ extension AdaptiveWindowTypeExtension on AdaptiveWindowType {
         throw AssertionError('Unsupported AdaptiveWindowType');
     }
   }
+
+  bool operator <=(Object other) {
```

Comment:
I think these operations would be easier if you put the types into map to integers and compared those numbers instead (this way you don't rely on ordering of enum)

Reply:
I think its easier using the switch statements for readers. I do think if we were dealing with more enums this would be a simpler solution but I think it might be better as is.